cmd/root.go

```diff
@@ -46,11 +48,13 @@ Each namespace will have a PolicyReport with the outcome of the scan for resourc
 There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			level.SetZeroLogLevel()
 			namespace, err := cmd.Flags().GetString("namespace")
 			if err != nil {
 				return err
 			}
+			slogger = slog.New(logconfig.NewSlogger(os.Stdout, level))
+			slog.SetDefault(slogger)
```

Comment:
this is not needed, we are not using the default logger anywhere

Reply:
Technically that might be true, but we were setting log level at global level, no?
```go
func (l *Level) SetZeroLogLevel() {
	level, err := zerolog.ParseLevel(l.String())
	if err != nil {
		zerolog.SetGlobalLevel(zerolog.InfoLevel)
	} else {
		zerolog.SetGlobalLevel(level)
	}
}
```
I have removed it. But it's causing a few issues, e.g., need to import `log` and `slog` in this file, which is yelled at by the linter.